SortInventory/SortInventoryMod.cs

```diff
@@ -122,4 +123,22 @@ public void OnModUnload()
             Debug.Log("SortInventory has been unloaded!");
         }
     }
-}
\ No newline at end of file
+
+    sealed class ItemComparer : IComparer<ItemInstance>
+    {
+        public int Compare(ItemInstance x, ItemInstance y)
+        {
+            var localized = String.Compare(x.settings_Inventory.DisplayName, y.settings_Inventory.DisplayName, StringComparison.CurrentCultureIgnoreCase);
```

Comment:
`ItemInstance.settings_Inventory.DisplayName` is not properly changed when updating the language. Use `ItemInstance.baseItem.settings_Inventory.DisplayName` instead.

Reply:
Got it. I'll update this.